### PR TITLE
Mbm 385 magic slots

### PR DESCRIPTION
**Pull request by insertStringHere**

### CODE REVIEW

File: src/main/java/com/mineboundteam/minebound/capabilities/ArmorSpellsProvider.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        @Override
+        public <T> LazyOptional<T> getCapability(Capability<T> cap, Direction side) {
+            return cap == ARMOR_ACTIVE_SPELLS ? optional.cast() : LazyOptional.empty();
+        }
+    }
+
+    public static class ArmorPassiveSpellsProvider extends ArmorSpellsProvider {
+        @Override
+        public <T> LazyOptional<T> getCapability(Capability<T> cap, Direction side) {
+            return cap == ARMOR_ACTIVE_SPELLS ? optional.cast() : LazyOptional.empty();
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/capabilities/ArmorSpellsProvider.java:68]: Should this be `ARMOR_PASSIVE_SPELLS`?

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/capabilities/ArmorSpellsProvider.java:68]: Yes

**Comment by MattKirchh** [comment]
Is it too much to ask to widen this UI pane? Feels too tightly packed with "Passive Magic".
image

**Comment by MattKirchh** [comment]
If I use say a Myrial Effigy piece with a spell in it to craft a Myrial Suit piece, will that spell persist with the newly crafted armor piece with these changes?

**Comment by insertStringHere** [comment]
> If I use say a Myrial Effigy piece with a spell in it to craft a Myrial Suit piece, will that spell persist with the newly crafted armor piece with these changes?

It should, that's the code in the anonymous ResultsContainer. I haven't' tested it, though, because we don't have those recipes in yet. If it needs fixing, whoever does the recipes can fix it; I doubt it'll be a difficult change.

**Comment by insertStringHere** [comment]
> Is it too much to ask to widen this UI pane? Feels too tightly packed with "Passive Magic". image

It would require the slots to also be moved to accommodate the new size. I changed the text to Utility Magic; does that work better?

**Comment by MattKirchh** [comment]
> > Is it too much to ask to widen this UI pane? Feels too tightly packed with "Passive Magic". image
> 
> It would require the slots to also be moved to accommodate the new size. I changed the text to Utility Magic; does that work better?

Yeah that looks a lot better.

**Review by MattKirchh — APPROVED** [review]
As far as I can tell this looks great. Great work.

**Review by solidwaterslayer — APPROVED** [review]

File: src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public class BlockRegistry {
     public static final RegistryObject<Block> ALLOY_FURNACE = BLOCKS.register("alloy_furnace", () -> new AlloyFurnace(Properties.copy(Blocks.FURNACE)));
     public static final RegistryObject<Block> ARMOR_FORGE = BLOCKS.register("armor_forge", () -> new ArmorForge(Properties.copy(Blocks.ANVIL)));
     public static final RegistryObject<Block> ENERGIZED_IRON_BLOCK = BLOCKS.register("energized_iron_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK)));
-    public static final RegistryObject<Block> ILLUMINANT_STEEL_BLOCK = BLOCKS.register("illuminant_steel_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK).strength(7, 9)));
+    public static final RegistryObject<Block> ILLUMINANT_STEEL_BLOCK = BLOCKS.register("illuminant_steel_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK).strength(7, 9).lightLevel(i -> 13)));
```
Comment (YonatanWN) [inline src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java:30]: Why is this in here?

**Review by YonatanWN — APPROVED** [review]

**Pull request merged**